tests/test_storage.py

```diff
@@ -16,6 +16,9 @@
 
 
 def test_postgres():
+    if os.getenv("OPENAI_API_KEY") is None:
```

Comment:
@sarahwooders @vivi I think this should stop tests from failing when they're from a fork